pythonforandroid/toolchain.py

```diff
@@ -204,9 +208,10 @@ def build_dist_from_args(ctx, dist, args):
                   ignore_project_setup_py=getattr(
                       args, "ignore_setup_py", False
                   ),
+                  debug_build=(args.build_mode == "debug"),
                  )
 
-    ctx.bootstrap.run_distribute()
+    ctx.bootstrap.run_distribute(debug_build=(args.build_mode == "debug"))
```

Comment:
Minor: if it's the second time we use such `debug_build` as a boolean, maybe we could create a temp variable.
```python
debug_build = args.build_mode == "debug"
```